src/redux/hooks/transactionsHooks.ts

```diff
@@ -0,0 +1,17 @@
+import { useAppSelector } from '@redux/hooks/appHooks';
+import { TRANSACTION_NAMESPACE, ITransactionState } from '@redux/reducers/transactionReducer';
+
+// eslint-disable-next-line
+export function useTransactionSelector<T extends (_state: ITransactionState) => any>(
```

Comment:
The same as above

Reply:
I just fixed it.
This commit
https://github.com/pastelnetwork/pastel-explorer-frontend/pull/110/commits/85258d89013d4fa0294078643721f092e9528ebf